index.html

```diff
@@ -1,56 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Awesome Books</title>
-    <link rel="stylesheet" href="index.css">
-</head>
-<body>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
     <header>
-        <nav>
-            <h2 id="title">Awesome Books</h2>
-            <ul id="nav-buttons">
-                <li><button id="list-btn" class="navbtn border-btn">List</button></li>
-                <li><button id='add-btn' class="navbtn border-btn">Add New</button></li>
-                <li><button id='contact-btn' class="navbtn">Contact</button></li>  
-            </ul>
-        </nav>
+      <nav>
+        <h2 id="title">Awesome Books</h2>
+        <ul id="nav-buttons">
+          <li><button id="list-btn" class="navbtn border-btn">List</button></li>
+          <li>
+            <button id="add-btn" class="navbtn border-btn">Add New</button>
+          </li>
+          <li><button id="contact-btn" class="navbtn">Contact</button></li>
+        </ul>
+      </nav>
     </header>
 
     <main class="main-container">
-        <div id="date-container">
-            <p id="date"></p>
-        </div>
-        <div class="head-title">
-            <h1 id="section-title">Add a New Book</h1>
-        </div>
-        <div class="app-container">
-            <ul id="books" class="nodisplay">
-
-            </ul>
+      <div id="date-container">
+        <p id="date"></p>
```

Comment:
- [ ] Good job so far your project is well implemented :+1:  But Please add the date and time, they should be visible on your app from the project requirement you should use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6). Follow the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). :+1: 

![Screenshot from 2022-08-11 07-34-14](https://user-images.githubusercontent.com/6615394/184087337-ab6198aa-905c-48d0-bfc5-7ba435009e6f.png)